src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dao/BeneficialOwnerGovernmentOrPublicAuthorityDao.java

```diff
@@ -125,6 +138,38 @@ public void setNonLegalFirmMembersNatureOfControlTypes(List<NatureOfControlType>
         this.nonLegalFirmMembersNatureOfControlTypes = nonLegalFirmMembersNatureOfControlTypes;
     }
 
+    public List<NatureOfControlType> getNonLegalFirmControlNatureOfControlTypes() {
+        return nonLegalFirmControlNatureOfControlTypes;
+    }
+
+    public void setNonLegalFirmControlNatureOfControlTypes(List<NatureOfControlType> nonLegalFirmControlNatureOfControlTypes) {
+        this.nonLegalFirmControlNatureOfControlTypes = nonLegalFirmControlNatureOfControlTypes;
+    }
+
+    public List<NatureOfControlType> getTrustNatureOfControlTypes() {
```

Comment:
the getter and setter will need renaming to include the 'Control' - getTrustControlNatureOfControlTypes and set..

Reply:
See question above.

Reply:
Updated